samples/Reader/20_Reader_worksheet_hyperlink_image.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use PhpOffice\PhpSpreadsheet\IOFactory;
+
+require __DIR__ . '/../Header.php';
+
+$inputFileType = 'Xlsx';
+$inputFileName = __DIR__ . '/sampleData/example3.xlsx';
```

Comment:
We try to avoid adding XLSX files when possible to avoid growing the size of the repository. So please delete `samples/Reader/sampleData/example3.xlsx`, and either change the sample to write and read, or delete the sample entirely (but do not keep it with only read).